src/Libraries/Nop.Data/Migrations/UpgradeTo460/DataMigration.cs

```diff
@@ -220,6 +221,19 @@ T getAttributeValue<T>(IList<GenericAttribute> attributes, string key, Func<stri
                     }
                 );
             }
+
+            //#5607
+            var permissionRecordTypeTable = _dataProvider.GetTable<PermissionRecord>();
+
+            if (!permissionRecordTypeTable.Any(prt => string.Compare(prt.SystemName, "MultiFactorAuthentication", StringComparison.InvariantCultureIgnoreCase) == 0))
+                _dataProvider.InsertEntity(
+                    new PermissionRecord
+                    {
+                        SystemName = "ForceMultiFactorAuthentication",
+                        Name = "Public store. Force multi-factor authentication",
+                        Category = "PublicStore"
```

Comment:
Let's define a new category (`Security`) since this permission applies to both the public store and the admin area.